static/js/jira-configuration.js

```diff
@@ -232,11 +234,12 @@ if (disconnectOrgBtn != null) {
 	$(disconnectOrgBtn).click((event) => {
 		event.preventDefault();
 		const orgName = $(event.target).data("org-name");
-		const installationId = $(event.target).data("installation-id");
+		const gitHubInstallationId = $(event.target).data("installation-id");
+		const appId = $(event.target).data("app-id");
 		const modalTitle = `Disconnect ${orgName}?`;
 		const modalInfo = `Are you sure you want to disconnect your organization ${orgName}? This means that you will have to redo the backfill of historical data if you ever want to reconnect.`;
 		const disconnectType = "org";
-		handleModalDisplay(modalTitle, modalInfo, disconnectType, installationId);
+		handleModalDisplay(modalTitle, modalInfo, disconnectType, gitHubInstallationId, appId);
```

Comment:
Instead of adding another parameter, can we make `disconnectData` as an object instead? 
```suggestion
		mapDisconnectRequest(disconnectType, JSON.stringify({ gitHubInstallationId, appId}));
```
In that way, we don't have to keep adding multiple parameters if needed in future. We simply append this with the rest of the body params(`jiraHost`, `jwt`) in `mapDisconnectRequest` and pass it along for the _DELETE_ request.

Reply:
I can change it to an object but we need to pass generic property names because handleModalDisplay is used to append data to the modal and that needs to be able to easily access the data.

Reply:
If we pass in the names `installationId` and `appId`, which is exactly how backend expects, then we can simply merge it, isn't it? Like in the screenshot above.